pytradfri/gateway.py

```diff
@@ -150,6 +151,36 @@ def process_result(result):
         return Command('get', [ROOT_SMART_TASKS, task_id],
                        process_result=process_result)
 
+    def reboot(self):
+        """
+        Reboot gateway.
+
+        Returns a Command.
+        """
+        def process_result(result):
```

Comment:
Do you need to wrap the result?

Reply:
I didnt' succeed in getting any response from the GW. Not idea how to provoke a response (namely negative response). That's why I put the wrapper in there but left it empty in the end. 

Should I remove them?

Reply:
Yeah, it doesn't add anything.